Fix issues in `network-interaction.md` (#3680)
ballerina-platform/ballerina-dev-website: learn/guides/distinctive-language-features/network-interaction.md

The second is a Listener object. It receives network input and makes calls to remote methods of attached service objects. Service objects are attached to Listeners, which are registered with a module that is the third key entity in providing network services.

> Modules have a lifecycle and are initialized on program startup. Post initializations, the registered listeners startup, and when the program shuts down, the registered listeners also shut down.

Reviewer: The previous form (the verb) was correct

```suggestion
Modules have a lifecycle and are initialized on program startup. Post initialization, the registered listeners start up, and when the program shuts down, the registered listeners also shut down.
```

Or we should say
```suggestion
Modules have a lifecycle and are initialized on program startup. Post initialization, the registered listeners are started up, and when the program shuts down, the registered listeners are also shut down.
```

As mentioned earlier, all modules have a lifecycle, so let's understand a few aspects of the lifecycle.

> During initialization, all module listeners are registered. The initialization sequence is ordered such that if module *A* imports module *B*, then module *A* is initialized after module *B*. The initialization phase ends by calling the ``main`` function if present.

Reviewer: ```suggestion
During initialization, all module listeners are registered. The initialization sequence is ordered such that if module ``A`` imports module ``B``, then module ``A`` is initialized after module ``B``. The initialization phase ends by calling the ``main`` function if present.
```

string greeting = obj.greet();
```

> In the above code example, the ``object`` keyword is used in the expression that initializes the **``obj``** variable. This is called an object constructor. It has one method **``greet()``** that returns a string *"Hello World"*. So there is no class involved in this case.

Reviewer: ```suggestion
In the above code example, the ``object`` keyword is used in the expression that initializes the **``obj``** variable. This is called an object constructor. It has one method **``greet()``** that returns a string ``"Hello World"``. So there is no class involved in this case.
```
